src/test/resources/features/offender_adjudications.feature

```diff
@@ -14,7 +14,7 @@ Feature: Offenders Adjudications
     And the associated agencies for this offender are: "LEI,MDI"
 
   Scenario: A staff user views many adjudications for an existing offender
-    Given a user has a token name of "SYSTEM_USER_READ_WRITE"
```

Comment:
Have just swapped these tests to use an alternative override role (rather than rewriting tests) as this functionality has now been removed from Nomis.  All calling clients will shortly be moving over to the DPS service and so these tests can then be removed.